CDTDatastore/CDTDatastore.h

```diff
@@ -54,12 +54,8 @@ extern NSString * __nonnull const CDTDatastoreChangeNotification;
  * propagated, thereby resolving the conflicted document across the set of
  * peers.
  *
- * **WARNING:** conflict resolution is coming in the next
- * release, where we'll be adding methods to:
- *
- * - Get the IDs of all conflicted documents within the datastore.</li>
- * - Get a list of all current revisions for a given document, so they
- *     can be merged to resolve the conflict.</li>
+ * See CDTDatastore+Conflicts.h for functions to resolve Document conflicts caused by
```

Comment:
I'm happy this fixes the issue mentioned, but I also notice the paragraph before this mentions `[CDTDatastore deleteDocumentWithId:rev:error:]`, which doesn't exist, so that paragraph should probably be corrected while we're here.